.github/workflows/controllo_link.yml

```
@@ -0,0 +1,63 @@
+name: Broken Links Checker
+on:
+  push:
+    branches:
+      - master
+  schedule:
+     - cron: '0 0 1 * *'
+env:
+  ISSUE_TEMPLATE: ".github/workflows/ISSUE_TEMPLATE.md"
+ 
+  contents: read
+  issues: write 
+  
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get links from README.md
+      id: get-links
+      run: |
+       LINKS=$(grep -oP '\[.*?\]\((http[s]?://[^)]*)\)' README.md | sed -E 's/\[.*\]\(([^)]+)\)/\1/' | paste -sd "," -)
```

```suggestion
       LINKS=$(grep -oP '\[.*?\]\((http[s]?://[^)]*)\)' README.md | sed -E 's/\[.*\]\(([^)]+)\)/\1/' | paste -sd ,)
```

Can't we use just `sed` here?